src/JSFunctions/fetchData.js

```diff
@@ -0,0 +1,32 @@
+const fetchData = async () => {
+  const listContaier = document.getElementById('list');
+  const getData = await fetch('https://themealdb.com/api/json/v1/1/filter.php?c=Seafood')
+    .then((res) => res.json())
+    .then((data) => data.meals);
+  // Create The List
+  return getData.map((item) => {
+    listContaier.innerHTML += `
```

Comment:
I think it would be nice to put the lines below in a function and call it in the `fetchdata` function. That way it would look better structured and make it easier for testing.
https://github.com/ahmed-al-farouq/Javascript-capstone/blob/c2e573c8798c277ce7c485d8a00eee47e471e16b/src/JSFunctions/fetchData.js#L8-L26

Reply:
Thanks.